ingress/src/main/java/com/redhat/developer/ingress/IngressService.java

```diff
@@ -0,0 +1,13 @@
+package com.redhat.developer.ingress;
+
+import io.cloudevents.CloudEvent;
+
+public interface IngressService {
+
+    void processEvent(String name, CloudEvent event);
+
+    // TODO: remove after we move to k8s
+    String deploy(String name);
```

Comment:
@danielezonca It is a mock because the `Ingress` application will be deployed by k8s according to a specific `Deployment` or CRD. 
I expect that the "core" of the ingress application will remain the same: it will receive a cloud event on a specific endpoint and then it will push it to a kafka topic. I commented all the "infrastructural" parts that will be handed over to k8s, for example the deployment of the instance (in this class we keep a list of "deployed" instances, and if the user tries to push an event to an undeployed instance we return an error. When we move to k8s, we just delete all this logic because the application is deployed by k8s)

Reply:
ok thanks for the clarification 👍